docs/openapi/components/schemas/snippets/IssuerOrganization.yml

```diff
@@ -0,0 +1,112 @@
+title: Issuer Organization
+description: Organization issuing a Verifiable Credential.
+type: object
+properties:
+  type:
+    type: array
+    readOnly: true
```

Comment:
Doesn't `readOnly` in this context imply that this value must not be present when submitting a request that contains this type of object in the request body, but it will be present in responses containing this type of object?

Reply:
Hey Chris, no the contrary, this is to tell a form renderer that you got not choice but to include it. 
https://jsonforms.io/docs/readonly/

We do this everywhere for `type` and `@context`.